include/constants/narc.h

```diff
@@ -42,7 +42,7 @@ enum {
     NARC_INDEX_GRAPHIC__FIELD_BOARD,
     NARC_INDEX_FIELDDATA__ENCOUNTDATA__D_ENC_DATA,
     NARC_INDEX_GRAPHIC__PL_WINFRAME,
-    NARC_INDEX_GRAPHIC__PL_PST_GRA,
```

Comment:
**issue**: We usually keep these aligned with the actual path of the NARC; this would be a marked shift away from that standard. I can be convinced that that's the right direction moving ahead, but we should file a follow-up issue if we decide as-such.

Reply:
I was thinking it might be nice for it to have an actual name since the file can't have one, but the original name does more clearly convey that it's associated with that NARC file. I switched it back.